syntax = "proto3";

package foo;

message FooRequest {
    int32 index = 1;
    string message = 2;
}
message FooResponse {
    int32 index = 1;
    string message = 2;
}
service FooGrpc {
    rpc getFoo(FooRequest) returns (FooResponse) {}
}
